src/plugins/ui_stateful/sheetview.ts

```diff
@@ -640,6 +643,18 @@ export class SheetViewPlugin extends UIPlugin {
     return CommandResult.Success;
   }
 
+  private checkViewportsWillChange({ offsetX, offsetY }: SetViewportOffsetCommand) {
```

Comment:
Nitpickest : I don't like this name but I don't have any better idea ... Maybe `CheckViewportsChange(s)` or `CheckIfViewportsWillChange` ?